core/src/main/java/org/mule/api/MuleException.java

```diff
@@ -199,9 +203,14 @@ public String getSummaryMessage()
         }
         StringBuilder buf = new StringBuilder(1024);
         buf.append(SystemUtils.LINE_SEPARATOR).append(EXCEPTION_MESSAGE_DELIMITER);
+        Pattern p = compile("passphrase=([^,])+");
```

Comment:
Pass this logic to a method method in CredentialsMaskUtils.
Use a constant for the regex passphrase=([^,])+